docs/configuration/fault-remediation.md

```diff
@@ -0,0 +1,246 @@
+# Fault Remediation Configuration
+
+## Overview
+
+The Fault Remediation module creates maintenance Custom Resources (CRs) that trigger external repair systems to fix faulty nodes. This document covers all Helm configuration options and extension points for system administrators.
+
+## Configuration Reference
+
+### Module Enable/Disable
```

Comment:
nit: Can we clarify that this module cannot be enabled independently, as DB, ND and FQ also need to be enabled?

What do you think?